kolibri/plugins/device/assets/src/views/ManageContentPage/NewBadge.vue

```diff
@@ -0,0 +1,40 @@
+<template>
+
+  <span
+    class="new-label"
+    :style="{ color: $themeTokens.textInverted, backgroundColor: $themeTokens.success }"
+  >
+    {{ label }}
+  </span>
+
+</template>
+
+
+<script>
+
+  import commonDeviceStrings from '../commonDeviceStrings';
+
+  export default {
+    name: 'NewBadge',
+    mixins: [commonDeviceStrings],
+    props: {
+      label: {
+        type: String,
+        required: true,
+      },
+    },
+  };
+
+</script>
+
+
+<style lang="scss" scoped>
+
+  .new-label {
```

Comment:
This `new-label` class definition has moved _inside_ the `NewBadge` component but has conflicting definitions that remain in parent components.

Classes used on the root node of a child component are a subtle case where vue's scoped styles mechanism breaks down, and it's possible for class definitions to collide with between child and parent component styles.

See https://vue-loader.vuejs.org/guide/scoped-css.html#child-component-root-elements

Therefore, it's important that the class name used inside the badge components scoped styles is different than class names defined and applied from outside the badge.

Reply:
Hi @indirectlylit!
Thanks for the information, it was new to me.
I have only put the common stylings - the 'badge specific' styles like padding and font size - in the `NewBadge` component, as should be the case. The stylings for its positioning relative to the external components are defined in the parent components. Hence the same class name won't create conflicts but compliment each other. 

But I agree with you, the parent and child shouldn't be this tightly coupled. I have changed the class name of the component. Please take a look.
Thanks!

Reply:
thank you! that addresses my concern.

I haven't tested locally and will let @jonboiser or others give the final OK to merge